client/src/organisms/RoomSearch.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
+    <div class="inline pb-10">
+      <DatePicker v-model="startDate" pickerId="start-date-picker" />
+      <div class="centered">bis</div>
+      <DatePicker v-model="endDate" pickerId="end-date-picker" />
+    </div>
+    <MainButton identifier="search" @search="search" />
+    <FeedbackMessage
+      :isOpen="isAlertOpen"
+      message="End date cannot be before start date"
+      @close-alert="isAlertOpen = false"
+    />
+  </div>
+</template>
+
+<script>
+import { MainButton } from "@/atoms";
+import { DatePicker } from "@/molecules";
+import { useRoomStore } from "@/store";
+import FeedbackMessage from "@/atoms/FeedbackMessage.vue";
+
+export default {
+  name: "RoomSearch",
+  components: {
+    DatePicker,
+    MainButton,
+    FeedbackMessage,
+  },
+  data() {
+    return {
+      startDate: new Date().toISOString(),
+      endDate: new Date().toISOString(),
+      isAlertOpen: false,
+      roomStore: useRoomStore(),
+    };
+  },
+  methods: {
+    search() {
+      if (this.startDate && this.endDate) {
+        if (new Date(this.endDate) < new Date(this.startDate)) {
+          this.isAlertOpen = true;
+        } else {
+          this.roomStore.fetchAvailableRooms(this.startDate, this.endDate);
```

Comment:
Please Change to the single endpoint like mentioned somewhere else to only have one endpoint for requesting rooms which takes all arguments